### PR TITLE
Homework2 : Dmitry Makarevich

### PR DESCRIPTION
**Pull request by DzmitryMak**
### PR checklist
Please make sure you've passed PR checklist
- [ ] my PR passes CI checks (flake8 and pycodestyle).
- [ ] my PR named according to guidelines.
- [ ] my PR contains complete tasks descriptions for all tasks.
- [ ] my PR made from the fork feature branch to appropriate branch.
- [ ] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore.

### CODE REVIEW

File: src/homework2/task2.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Учтите что в предложении есть знаки препинания.
+Подсказки:
+-	my_string.split([chars]) возвращает список строк.
+-	len(list) - количество элементов в списке
+"""
+
+my_string = input('Введите предложение: ')
+
+larg_word = ''
+punctuation_marks = ',.!:-?№/'
```
Comment (belovmd) [inline src/homework2/task2.py]: what about `]);` and others

File: src/homework2/task5.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+циклические операторы и условные операторы. n - вводится
+"""
+
+n = int(input('введите порядковый номер числа Фибоначчи  '))
+
+if n > 2:
+    num_f = num_s = sum_ = 1
+    i = 2
+    while i < n:
+        sum_ = num_s + num_f
```
Comment (belovmd) [inline src/homework2/task5.py]: a, b = a+b, a

**Review by belovmd — CHANGES_REQUESTED** [review]

File: src/homework2/task2.py (hunk 1 above)
Comment (DzmitryMak) [reply src/homework2/task2.py]: fixed

File: src/homework2/task5.py (hunk 2 above)
Comment (DzmitryMak) [reply src/homework2/task5.py]: fixed

File: src/homework2/task2.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+Учтите что в предложении есть знаки препинания.
+Подсказки:
+-	my_string.split([chars]) возвращает список строк.
+-	len(list) - количество элементов в списке
+"""
+
+my_string = input('Введите предложение: ')
+
+larg_word = ''
+punctuation_marks = ',.!:-?№/[]();'
```
Comment (belovmd) [inline src/homework2/task2.py:12]: ' " and others
review not addressed

**Review by belovmd — APPROVED** [review]
9

**Pull request closed without merging**